example/targetcolor/example_TargetColor.py

```diff
@@ -32,7 +32,7 @@ def setup_class(
             output_path='targetcolor',
         )
         # Setup matplotlib backend
-        matplotlib.use('TkAgg')
+        # matplotlib.use('TkAgg')

```

Comment:
Feel free to delete this line. I did a similar thing in figure_management.py a while ago:

```python
# matplotlib.use('tkagg') # BGB I don't think we should depend on this
mnger = plt.get_current_fig_manager()
if hasattr(mnger, 'window'):
    if hasattr(mnger.window, 'wm_geometry'):
        mnger.window.wm_geometry(f"+{ul_x}+{ul_y}")
    elif hasattr(mnger.window, 'geometry'):
        curr_dims = mnger.window.geometry().getRect()  # x, y, w, h
        mnger.window.setGeometry(curr_dims[0], curr_dims[1], ul_x, ul_y)
```